src/App/Pages/Accounts/LoginPasswordlessPage.xaml.cs

```diff
@@ -24,8 +24,21 @@ private async void Close_Clicked(object sender, System.EventArgs e)
         {
             if (DoOnce())
             {
+                _vm.StopRequestTimeUpdater();
```

Comment:
If I'm not mistaken we don't need this anymore given that closing the view will call `OnDisappearing` but test it just in case I'm missing something